.golangci.yml

```diff
@@ -1,3 +1,6 @@
+run:
+  timeout: 8m
```

Comment:
Bumped this as the linter was timing out while running. We got too much code now 😐